apps/backend/src/db/migrations/20260722120000_thread_anchor_cleanup.sql

```
@@ -0,0 +1,22 @@
+-- Event-anchored threads (cleanup, Phase 2): drop the grace-period legacy shapes.
+--
+-- The substrate migration (20260721120000) added `parent_anchor_id` and moved
+-- reply stats onto the thread stream row, keeping `streams.parent_message_id`,
+-- its unique index, and `messages.reply_count` alive so old-deploy replicas
+-- stayed correct during the rollout. All code now reads/writes only the anchor
+-- track, so the legacy columns are dead. This migration deploys AFTER chunk 1-6
+-- code is everywhere (merge is gated for exactly that reason): dropping the
+-- column while an old-old replica still reads it would break that replica.
+
+-- Drop the legacy per-message anchor index before the column it references.
+DROP INDEX IF EXISTS idx_streams_thread_parent;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)